types/sphere-engine-browser/index.d.ts

```diff
@@ -0,0 +1,3001 @@
+// Type definitions for the non-npm package sphere-engine-browser 5.5
```

Comment:
This will be published as `@types/sphere-engine-browser@5.5.X`. What is this number? You previously described this as the “v2 API,” so I was expecting this number to start with 2.

Reply:
5.5 is the current miniSphere version, I figured it would make sense to have the package follow the engine version when changes are made to the API. And it's referred to as the "v2" API in an abstract sense. There had been different attempts at making an official engine update, since the last official Sphere release was v1.5, with an unfinished 1.6 beta (using the v1.x API)